pom.xml

```diff
@@ -104,7 +109,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>jackson2-api</artifactId>
-      <version>2.12.1</version>
+      <version>2.13.3-285.vc03c0256d517</version>
```

Comment:
why are we not just using a more recent bom version that contains this?
e.g. https://github.com/jenkinsci/bom/blob/807.v6d348e44c987/bom-latest/pom.xml  (newer are available)

Reply:
you're right, will clean this all up